src/handlers.ts

```diff
@@ -108,13 +112,36 @@ export const postFeedback = async (ctx: IRouterContext): Promise<any> => {
     throw { status: 401 }
   }
 
+  const [feedback] = await db<Feedback>('feedbacks').insert({ status, user_id: user.id }).returning('*')
```

Comment:
As discussed, let's grab the encodings for each of the files so we can do this in one transaction

Reply:
I'm not sure that inserting the encoding would work. Why not insert a binary?